src/aws/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM busybox as mediainfo
+
+# Embed the mediainfo layer in the image
+RUN wget https://mediaarea.net/download/binary/mediainfo/20.09/MediaInfo_CLI_20.09_Lambda.zip && \
+    unzip MediaInfo_CLI_20.09_Lambda.zip -d /tmp/mediainfo
+
+FROM amazon/aws-lambda-nodejs:12 as core
+
+# COPY mediainfo in core stage
+COPY --from=mediainfo /tmp/mediainfo/bin/mediainfo /opt/bin/mediainfo
+
+# Copy sources
+# LAMBDA_TASK_ROOT is defined in base image and its value is /var/task
+COPY lambda-complete ${LAMBDA_TASK_ROOT}/lambda-complete/
+COPY lambda-configure ${LAMBDA_TASK_ROOT}/lambda-configure/
+COPY lambda-encode ${LAMBDA_TASK_ROOT}/lambda-encode/
+COPY lambda-medialive ${LAMBDA_TASK_ROOT}/lambda-medialive/
+COPY lambda-medialive-routing ${LAMBDA_TASK_ROOT}/lambda-medialive-routing/
+COPY lambda-migrate ${LAMBDA_TASK_ROOT}/lambda-migrate/
+COPY utils ${LAMBDA_TASK_ROOT}/utils/
+
+# Install yarn and yarn-recursive
+RUN npm install -g yarn
+
+# Image used in developement with development dependencies
+FROM core as development
+
+RUN cd lambda-complete && yarn install && \
+    cd ../lambda-configure && yarn install && \
+    cd ../lambda-encode && yarn install && \
+    cd ../lambda-medialive && yarn install && \
+    cd ../lambda-medialive-routing && yarn install && \
+    cd ../lambda-migrate && yarn install
```

Comment:
By doing so, you will almost never use the docker cache for this step. Is this the desired behavior?